ArraysHomework/ArraysHomework.cs

```diff
@@ -157,27 +206,46 @@ public void Lists()
             // BONUS reverse this list
             List<int> reverseMe = new List<int>() { 1, 2, 3, 4, 5 };
 
-            //Assert.IsTrue(reverseMe[0] == 5);
-            //Assert.IsTrue(reverseMe[1] == 4);
-            //Assert.IsTrue(reverseMe[2] == 3);
-            //Assert.IsTrue(reverseMe[3] == 2);
-            //Assert.IsTrue(reverseMe[4] == 1);
+            reverseMe.Reverse();
+
+            Assert.IsTrue(reverseMe[0] == 5);
+            Assert.IsTrue(reverseMe[1] == 4);
+            Assert.IsTrue(reverseMe[2] == 3);
+            Assert.IsTrue(reverseMe[3] == 2);
+            Assert.IsTrue(reverseMe[4] == 1);
 
             // ---------
             
             // BONUS sort this list
             List<int> sortMe = new List<int>() { 14, 3, 33, 2, 12 };
 
-            //Assert.IsTrue(sortMe[0] == 2);
-            //Assert.IsTrue(sortMe[1] == 3);
-            //Assert.IsTrue(sortMe[2] == 12);
-            //Assert.IsTrue(sortMe[3] == 14);
-            //Assert.IsTrue(sortMe[4] == 33);
+            sortMe.Sort();
+
+            Assert.IsTrue(sortMe[0] == 2);
+            Assert.IsTrue(sortMe[1] == 3);
+            Assert.IsTrue(sortMe[2] == 12);
+            Assert.IsTrue(sortMe[3] == 14);
+            Assert.IsTrue(sortMe[4] == 33);
 
             // ----------
 
             // BONUS Convert sortMe to an array
 
+            List<string> sortMe = new List<string>;
+
+            List.Add(sortMe[0]);
+            List.Add(sortMe[1]);
+            List.Add(sortMe[2]);    
+            List.Add(sortMe[3]);
+            List.Add(sortMe[4]);
+
+
+            string[] array = List.ToArray();
```

Comment:
This is very close.
`List` (capital L) is a type. (Specifically, now that we've gone over it, it's a class). You need to do something with the variable (again, to use the terminology we have now, but didn't have when we talked about arrays/Lists, you need to do something with the instance), not with the type `List`.

You're redefining the `sortMe` variable above here on line 234 (`List<string> sortMe = new List<string>`), which isn't allowed and might be messing with Intellisense (Visual Studio's suggestions). `sortMe` already exists and doesn't need to be created again.